website/docs/d/route53_record.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Route 53"
+layout: "aws"
+page_title: "AWS: aws_route53_record"
+description: |-
+  Terraform data source for managing an AWS Route 53 Record.
+---
+
+# Data Source: aws_route53_record
+
+`aws_route53_record` provides details about a records present in Route 53 Hosted Zone.
+
+This data source allows to find a Route53 records given Hosted Zone id and certain search criteria.
```

Comment:
If it can return multi-records in the same data_resource maybe the name of the resource should be renamed from `route53_record` to `route53_records`